RustyOptions.FSharp depends on Prerelease FSharp.Core

**Describe the bug**
When built on a pre-release version of .NET, RustyOptions.FSharp depends on a prerelease version of FSharp.Core instead of the current released version.

**To Reproduce**
`dotnet pack --configuration Release`

> /usr/local/share/dotnet/sdk/7.0.200-preview.22628.1/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(221,5): warning NU5104: A stable release of a package should not have a prerelease dependency. Either modify the version spec of dependency "FSharp.Core [7.0.1-beta.22616.1, )" or ]

**Expected behavior**
Should not get the above warning or reference a pre-release FSharp.Core

--- a/src/RustyOptions.FSharp.Tests/RustyOptions.FSharp.Tests.fsproj
+++ b/src/RustyOptions.FSharp.Tests/RustyOptions.FSharp.Tests.fsproj
@@ -27,6 +27,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Remove="FSharp.Core" />
+    <PackageReference Include="FSharp.Core" Version="7.0.0" />
   </ItemGroup>
 
   <ItemGroup>

--- a/src/RustyOptions.FSharp/RustyOptions.FSharp.fsproj
+++ b/src/RustyOptions.FSharp/RustyOptions.FSharp.fsproj
@@ -27,4 +27,9 @@
   <ItemGroup>
     <None Include="../../README.md" Pack="true" PackagePath="\" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Remove="FSharp.Core" />
+    <PackageReference Include="FSharp.Core" Version="7.0.0" />
+  </ItemGroup>
 </Project>

